tst/worksheets/general.expected/title.xml

```diff
@@ -6,6 +6,6 @@
     General Test
   </Title>
   <Date>
-    2018 August 30
+    30 August 2018
```

Comment:
I honestly do not like the way this is displayed now (but also not the way it was displayed before). I think the most natural way I can think of for me seems `August 30, 2018`. But this is probably an opinion and bikeshedding, so we can leave it this way. It is probably better than the old way already.

Reply:
Actually, that's the difference between wrong (as it was), british (the way it is now) and american (the way you prefer). See also e.g. https://www.englishclub.com/vocabulary/time-date.htm

Reply:
Okay, that explains it quite well. I am just use to see the American formatting all the time. But as said, I do not care too much here, I just wanted to point it out. Lets merge this :)